Public/Src/Cache/ContentStore/Hashing/ContentHasher.cs

```diff
@@ -387,10 +385,9 @@ private Task<bool> PushHashBufferAsync(byte[] buffer, int offset, int count)
 
             private void TransformBlock(byte[] buffer, int offset, int count, byte[] outputBuffer, int outputOffset)
             {
-                var start = _sw.ElapsedTicks;
+                Stopwatch sw = Stopwatch.StartNew();

```

Comment:
I would suggest using StopwatchSlim instead, because Stopwatch.StartNew allocates in the heap on each call.

Reply:
StopwatchSlim is defined in ContentStore.Library, a dependent of ContentStore.Hashing.